packages/react/src/Breadcrumbs/Breadcrumbs.module.css

```
@@ -9,6 +10,100 @@
   margin-bottom: 0;
 }
 
+[data-overflow='menu'] .BreadcrumbsList,
+[data-overflow='menu-with-root'] .BreadcrumbsList {
+  white-space: nowrap;
+  display: flex;
+  flex-direction: row;
+}
+
+.BreadcrumbsItem {
+  display: inline-grid;
+  grid-auto-flow: column;
+  align-items: center;
+  flex: 0 99999 auto;
+  min-width: auto;
+  font-size: var(--text-body-size-medium);
+  white-space: nowrap;
+  list-style: none;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:last-child {
+    .ItemSeparator {
+      display: none;
+    }
+  }
+}
+
+[data-overflow='menu'] .Item,
+[data-overflow='menu-with-root'] .Item {
+  display: inline-block;
+  font-size: var(--text-body-size-medium);
+  color: var(--fgColor-link);
```

```suggestion
  color: var(--fgColor-default);
```

I wasn't sure if we should include this styling change, but since we're adding the hover state we might as well.. it seems like the teams building their custom breadcrumb experiences are using the default text color anyway or overriding our link color.